src/js/components/VoterGuide/EditPositionAboutCandidateModal.jsx

```diff
@@ -84,17 +84,18 @@ export default class EditPositionAboutCandidateModal extends Component {
   }
 
   _onSupportStoreChange () {
-    var candidate_we_vote_id = this.props.position.ballot_item_we_vote_id;
-    this.setState({ supportProps: SupportStore.get(candidate_we_vote_id) });
+    var ballot_item_we_vote_id = this.props.position.ballot_item_we_vote_id;
```

Comment:
Hi @zachmonteith, we might be able to use a generalized EditPositionModal file, but I suspect we will need a different one for Candidates vs. Measures.